src/ngraph/pass/reshape_sinking.cpp

```diff
@@ -115,7 +116,7 @@ static void delete_reshape(shared_ptr<Node> reshape)
     NGRAPH_DEBUG << "Removing reshape " << reshape->get_name();
     if (!reshape->get_users().empty())
     {
-        ngraph::replace_node(reshape, reshape->get_argument(0));
+        ngraph::replace_node(reshape, reshape->get_argument(0), true);
```

Comment:
Use `reshape->input_value(0).get_node_shared_ptr()` or this will fail if you are reshaping something not from an output 0.

Reply:
@diyessi I think input_values() isn't there in r0.251